cmake/Modules/FindBLIS.cmake

```diff
@@ -0,0 +1,62 @@
+SET(BLIS_INCLUDE_SEARCH_PATHS
+ /usr/include
```

Comment:
Why this is needed? (I.e. is there a system where `/usr/include` is not in the search path)
```suggestion
```

Reply:
We added this following the convention of FindOpenBLAS.cmake as below :
SET(Open_BLAS_INCLUDE_SEARCH_PATHS
  **/usr/include**
  /usr/include/openblas
  /usr/include/openblas-base
  /usr/local/include
  /usr/local/include/openblas
  /usr/local/include/openblas-base
  /usr/local/opt/openblas/include
  /opt/OpenBLAS/include
  $ENV{OpenBLAS_HOME}
  $ENV{OpenBLAS_HOME}/include
)

I will remove this as suggested